xmovie/core.py

```diff
@@ -451,10 +514,13 @@ def save(
                     )
 
         # print frames
-        self.save_frames(dirname, progress=progress)
+        if parallel:
+            self.save_frames_parallel(dirname)
+        else:
+            self.save_frames_serial(dirname, progress=progress)
 
         # Create movie
-        write_movie(
+        combine_frames_into_movie(
```

Comment:
Great change!

Reply:
Thanks! I did my best to make everything more intuitive for my own sake when implementing this. Hopefully it'll help future contributors